### PR TITLE
Pollen dictionary

### PR DESCRIPTION
**Pull request by pddao**
- Project setup 
- Add several frontend pages 
- Add controller, service and repo for dictionary

### CODE REVIEW

**Review by yasminwaldeck — APPROVED** [review]
Sieht gut aus!

File: backend/src/main/java/de/pddao/backend/model/PollenItem.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class PollenItem {
+    private String englishName;
+    private String latinName;
+    private String beginSeason;
```
Comment (fabianschmauder) [inline backend/src/main/java/de/pddao/backend/model/PollenItem.java:15]: do you always have a month here? Maybe than use some thing like [Month](https://docs.oracle.com/javase/8/docs/api/java/time/Month.html)

File: frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (fabianschmauder) [inline frontend/README.md:3]: i would remove it or change it

File: frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "frontend",
```
Comment (fabianschmauder) [inline frontend/package.json:2]: you can also add prettier

File: frontend/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (fabianschmauder) [inline frontend/public/manifest.json:3]: you can also set your app name here

File: frontend/src/hooks/usePollenItems.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import * as apiservice from "../service/apiservice"
+
+export default function usePollenItems() {
+    const [pollenItems, setPollenItems] = useState([]);
+
+    useEffect(() => {
+        loadData()
+    }, [])
+
+    const loadData = () => {
```
Comment (fabianschmauder) [inline frontend/src/hooks/usePollenItems.js:11]: i would rename the function to load PollenData or something that more describes what happens

**Review by fabianschmauder — COMMENTED** [review]
A few comments 😉

**Pull request merged**